### PR TITLE
Bring back missing MOS files (closes #691)

### LINKED ISSUE #691: Missing .mos-files for several FMUs
Previously there have been .mos-files for generating FMUs for the following models which are used by OMSysIdent examples:

- HelloWorld.mos
- HelloWorldWithInput.mos
- Lin2DimODE.mos

These .mos-files seems to be missing in the present repo. For `HelloWorld` and `HelloWorldWithInputs` the model binaries are available, but I think one should also keep the sources.

The `Lin2DimODE`-FMU is (currently) only used in the example src/OMSysIdentLib/test_Lin2DimODE_cs_Fit.cpp which has no CI test coming with it. I could create a CI example using this FMU, so that it is not overlooked in future updates and create a PR to bring the mos files back.

### PR DESCRIPTION
**Pull request by bernhard-thiele**
### Related Issues

#691 

### Purpose

Bring back MOS files which got lost at some point in development.

### Approach

These are simply the slightly adapted MOS files from a previous repo state

### CODE REVIEW

File: testsuite/resources/HelloWorld.mos [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  parameter Real x_start = 1;
+  Real x(start=x_start, fixed=true);
+equation
+  der(x) = a*x;
+end HelloWorld;
+"); getErrorString();
+
+buildModelFMU(HelloWorld, version="2.0", fmuType="me_cs", fileNamePrefix="HelloWorld", platforms={"x86_64-linux-gnu"}); getErrorString();
+
+system("unzip HelloWorld.fmu -d HelloWorld/");
```
Comment (lochel) [inline testsuite/resources/HelloWorld.mos]: Please remove the sources inside the fmu.

```bash
system("zip -qd HelloWorld.fmu sources/* && unzip HelloWorld.fmu -d HelloWorld/");
```

Comment (bernhard-thiele) [reply testsuite/resources/HelloWorld.mos]: Hm, I first used this line (copied it form other files), but got an error when executing it on my machine. I will try once more

Comment (bernhard-thiele) [reply testsuite/resources/HelloWorld.mos]: Worked no without an error, whatever I did wrong the first time. I'll update the PR.

**Comment by bernhard-thiele** [comment]
Hm, the failing test has probably nothing to do with my changes. The `tlm1d.lua` complains "==240==ERROR: LeakSanitizer: detected memory leaks". Is this a glitch?

**Comment by lochel** [comment]
That is a glitch. After rerunning the tests, everything looks fine.

**Pull request merged**